### DISCUSSION TITLE
Web UI on Remote Server

### DISCUSSION THREAD

**Discussion by fatmazehraozdemir** [apache/airflow#39289] [Q&A]
I have installed Apache airflow on a remote server and run the command 'airflow webserver --port 9090'. When I connect to a browser on my local computer with http:://<server_ip>:9090, I cannot see the Web UI. What would be the reason?

**Comment by prakashpanditjain** [comment]
If you've started the Airflow webserver on a remote server using the command airflow webserver --port 9090, and you're unable to access the web UI from your local computer using http://<server_ip>:9090, there could be several reasons for this issue. Here's what you can check:

1. Firewall Settings:

Make sure that the port 9090 is open in the firewall on your remote server.
Check Airflow Configuration:

Ensure that Airflow is configured to listen on all IP addresses, not just localhost. You can check this in your airflow.cfg file. Look for the following line:
makefile

`web_server_host = 0.0.0.0`

2. Network Configuration:

Verify that your remote server's network configuration allows incoming connections on port 9090.
Check if there are any network configurations, like NAT, that might be preventing the connection.
Check Airflow Webserver Logs:

Look into the Airflow webserver logs for any errors or warnings that might indicate why the web UI is not accessible.

3. Verify URL:

Ensure that you're using the correct URL in your browser: http://<server_ip>:9090.
Restart Airflow Webserver:

Sometimes, restarting the Airflow webserver might solve the issue.
Once you've checked these points, you should be able to access the Airflow web UI from your local computer. If you're still facing issues, let me know, and we can troubleshoot further.
